applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -43,26 +45,67 @@ Syntax
 * The ``<apn>`` parameter is a string.
   It represents an alternative access point name (APN), other than the default primary APN, to use for downloading.
 
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+  #XFOTA: <primary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+  #XFOTA: <secondary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+
+* The ``<primary_partition_id>`` and ``<secondary_partition_id>`` integers give the partition ID.
+* The ``<MCUBOOT_version>`` integer. The value 1 corresponds to MCUboot versions 1.x.y.
+* The ``<img_size>`` integer gives the size of the application image in this partition in bytes.
+* The ``<img_version>`` string gives the version of the application image as "<major>.<minor>.<revision>+<build_number>".
+
+::
+
+  #XFOTA: <size>,<offset>
+
+* The ``<size>`` integer gives the size of the modem DFU area in bytes;
+* The ``<offset>`` integer gives the offset of the firmware image in the modem DFU area;
```

Comment:
Maybe full stop instead of the semicolons at the ends of these two.

Reply:
Applied, thanks